linkedList.js

```diff
@@ -4,5 +4,116 @@
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
 function linkedListGenerator(){
+  var head = null;
+  var tail = null;
+  var length = 0;
+  // var node = {
+  //           value: null,
+  //           next: {},
+  //         };
+  var linkedList = {
+    getHead: function(){
+              return head;
+            },
+    getTail: function(){
+              var thisNode = head;
+              if(!thisNode){
+                return null;
+              }
+              while(thisNode.next !== null){
+                thisNode = thisNode.next;
+              }
+              return thisNode;
+            },
+    add: function(value){
+            var newNode = {
+                  value: value,
+                  next: null
+                  };
+            if(head === null && tail === null){
+              head = newNode;
+              tail = newNode;
+            }else{
+              tail.next = newNode;
+              tail = newNode;
+            }
+            length++;
+            return newNode;
+         },
+    remove: function(number){
```

Comment:
As I mentioned earlier, I would recommend trying to use the `get` method to clean up this method.
🔧